osf/migrations/0138_ensure_subjects_and_providers.py

```diff
@@ -19,8 +19,6 @@
     'domain': DOMAIN,
     'share_publish_type': 'Preprint',
     'domain_redirect_enabled': False,
-    'default_license': 'CC0 1.0 Universal',
-    'licenses_acceptable': ['CC0 1.0 Universal', 'CC-By Attribution 4.0 International', 'No license'],
```

Comment:
(Applies to both Preprints and Registrations)

A comment for future reference: when we move the following in `0138_ensure_subjects_and_providers.py` to post-migration signal, can we add the `default_license` and `licenses_acceptable` back? This also depends on the execution order of post-migration signals where `update_license` needs to be run first.

```python
    operations = [
        migrations.RunPython(create_osf_registries, migrations.RunPython.noop),
    ] if getattr(settings, 'TEST_ENV', False) else [
        migrations.RunPython(create_osf_preprints, migrations.RunPython.noop),
        migrations.RunPython(create_subjects, migrations.RunPython.noop),
        migrations.RunPython(create_osf_registries, migrations.RunPython.noop),
    ]
```

I mentioned this because I don't think a provider is  a complete provider without a license. This also saves us the effort of adding license during testing.

```python
osf_provider.default_license = NodeLicense.objects.get(name='No license')
osf_provider.licenses_acceptable.add(NodeLicense.objects.get(name='No license'))
```